cime_config/machines/config_machines.xml

```diff
@@ -586,19 +586,136 @@
     </environment_variables>
   </machine>
 
+  <machine MACH="frontier">
+    <DESC>Frontier exascale supercomputer at ORNL. 9408 nodes, Node: 4 AMD MI250X GPUs (2 GCDs) ~ 8 GPUs, 512 GB HDB2E, AMD EPYC 64 cores, 512GB DDR4 </DESC>
+    <NODENAME_REGEX>.*frontier.*</NODENAME_REGEX>
+    <OS>CNL</OS>
+    <COMPILERS>gnu,crayclang,amdclang,gnugpu,crayclanggpu,amdclanggpu</COMPILERS>
+    <MPILIBS>mpich</MPILIBS>
+    <PROJECT>cli115</PROJECT>
+    <SAVE_TIMING_DIR>/lustre/orion/cli115/proj-shared/frontier</SAVE_TIMING_DIR>
+    <SAVE_TIMING_DIR_PROJECTS>.*</SAVE_TIMING_DIR_PROJECTS>
+    <CIME_OUTPUT_ROOT>/lustre/orion/cli115/proj-shared/$ENV{USER}/e3sm_scratch</CIME_OUTPUT_ROOT>
```

Comment:
How about removing the hard-coded project (`cli115`) and instead using `$PROJECT`? 
```
<SAVE_TIMING_DIR>/lustre/orion/$PROJECT/proj-shared/frontier</SAVE_TIMING_DIR>
<CIME_OUTPUT_ROOT>/lustre/$PROJECT/cli115/proj-shared/$ENV{USER}/e3sm_scratch</CIME_OUTPUT_ROOT>
```

Reply:
Sure, but we are only ever going to have `cli115` on Frontier.

Reply:
Anyway, they could probably use `--project` command line argument too. Let me think about it.

Reply:
>Anyway, they could probably use `--project` command line argument too. Let me think about it.

Yes, I'm not part of `cli115`, so I was using `--project <xyz>`. But, I had to additionally do two `./xmlchage` for `SAVE_TIMING_DIR` and `CIME_OUTPUT_ROOT`.

Reply:
> we still want the performance archive under cli115

In that case, how about instead using `/lustre/orion/cli115/world-shared`?

Reply:
I will change to world-shared and $PROJECT (CIME_OUTPUT_ROOT). 

Just curious, did land folks finally get a DD allocation on Frontier? We haven't run across the scenario where someone wasn't a member of cli115 (cli133 was a subset of folks and another _cli115_ecp_ is a virtual account under the same parent etc.)

Reply:
@bishtgautam What's your project ID so that I can ensure it's getting captured properly in the performance archive?